io/io/src/RRawFileUnix.cxx

```diff
@@ -115,8 +115,8 @@ void ROOT::Internal::RRawFileUnix::OpenImpl()
 void ROOT::Internal::RRawFileUnix::ReadVImpl(RIOVec *ioVec, unsigned int nReq)
 {
 #ifdef R__HAS_URING
-   if (RIoUring::IsAvailable()) {
-      RIoUring ring;
+   try {
+      RIoUring ring; // throws std::runtime_error
```

Comment:
I think it would be helpful for the RNTupleMetrics to expose the actual queue depth through RRawFile.

Reply:
Is it OK if I break this out into another PR?

Reply:
Yep, I think that makes sense

Reply:
We might need to change the `ReadV` logic such that it doesn't internally try to process all `nreqs` but only the ones it can do in parallel in one batch. And then it would return that number.